frontend/src/pages/ReviewWriting/components/KeywordButton/index.tsx

```diff
@@ -1,13 +1,17 @@
-import { PropsWithChildren } from 'react';
+import { ButtonHTMLAttributes, PropsWithChildren } from 'react';
 
 import * as S from './styles';
 
-interface KeywordButtonProps {
+interface KeywordButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
   isSelected: boolean;
 }
 
-const KeywordButton = ({ isSelected, children }: PropsWithChildren<KeywordButtonProps>) => {
-  return <S.KeywordButton isSelected={isSelected}>{children}</S.KeywordButton>;
+const KeywordButton = ({ isSelected, children, ...props }: PropsWithChildren<KeywordButtonProps>) => {
```

Comment:
```suggestion
const KeywordButton = ({ isSelected, children, ...props }: React.PropsWithChildren<KeywordButtonProps>) => {
```

PropsWithChildren import 하지 않아도 돼요~

Reply:
React에서 제공하는 타입으로 수정했어요 :)